lib/backend.ml

```diff
@@ -16,6 +16,8 @@
  * OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
  *)
 
+[@@@ocaml.warning "-3"]
```

Comment:
Would it be better to tag just the particular expression that is deprecated? This will hide all other deprecation warnings, if I understand correctly.